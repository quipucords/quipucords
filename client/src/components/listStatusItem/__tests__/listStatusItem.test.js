import React from 'react';
import { mount } from 'enzyme';
import ListStatusItem from '../listStatusItem';

describe('ListStatusItem Component', () => {
  it('should render', () => {
    const props = {
      key: 'credential',
      id: 'credential',
      count: 100,
      emptyText: '0 Credentials',
      tipSingular: 'Credential',
      tipPlural: 'Credentials',
      expanded: true,
      expandType: 'credentials',
      toggleExpand: jest.fn(),
      iconInfo: { type: 'fa', name: 'id-card' }
    };

    const component = mount(<ListStatusItem {...props} />);

    expect(component.render()).toMatchSnapshot();
  });
});
