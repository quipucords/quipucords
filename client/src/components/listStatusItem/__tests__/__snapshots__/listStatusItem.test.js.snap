// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListStatusItem Component should render 1`] = `
<div
  class="list-view-pf-additional-info-item list-view-info-item-icon-count"
>
  <span>
    <div
      class="list-view-pf-expand active"
    >
      <span
        class="fa fa-angle-right fa-angle-down"
      />
      <span
        aria-hidden="true"
        class="fa fa-id-card list-view-compound-item-icon"
      />
      <strong>
        100
      </strong>
    </div>
  </span>
</div>
`;
